src/playGame.js

```diff
@@ -0,0 +1,25 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RULES } from "../constants/messages.js";
+import randomNumber from "../utils/randomNumber.js";
+import userInputValidation from "../utils/userInputValidation.js";
+import resultPrint from "../utils/resultPrint.js";
+import compareToNumber from "../utils/compareToNumber.js";
+
+const playGame = async () => {
+  const comNum = randomNumber();
+  while (true) {
+    const userInput = await MissionUtils.Console.readLineAsync(
+      GAME_MESSAGES.INPUT
+    );
+    await userInputValidation(userInput);
+    const userNum = Array.from(userInput, Number);
+
+    const result = compareToNumber(comNum, userNum);
+    await resultPrint(result);
+    if (result[0] === RULES.NUM_LENGTH) {
+      break;
+    }
```

Comment:
```suggestion
    const [strike, ball, nothing] = compareToNumber(comNum, userNum);
    await resultPrint(result);
    if (strike === RULES.THREE_STRIKE) {
      break;
    }
```
여기서 `result`는 게임 카운트를 가지고 있고, `result[0]`이 스트라이크라서 게임 종료 조건이 걸려있는 것 같은데 약간 게임 요구사항을 모르면 `result[0]`이 무엇을 의미하는지 파악하기 어려울 것 같아요!

`compareToNumber`에서 반환받을 때 스프레드 문법으로 받아서 조금 더 명확하게 해줘도 좋을 것 같아요~!

그리고 `RULES.NUM_LENGTH`도 3이여서 재사용하신 것 같긴한데 사실 length 관련된게 조건에 있길래 3스트라이크라고 바로 생각은 못했던 것 같아요... 상수 `RULES`에서 `THREE_STRIKE` 항목을 추가하면 코드 파악에 도움이 될 것 같아요🙃